packages/fx-core/src/plugins/resource/bot/resources/strings.ts

```diff
@@ -1,22 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+import { getLocalizedString } from "../../../../common/localizeUtils";
+
 export class CommonStrings {
   public static readonly BOT_WORKING_DIR_NAME = "bot";
-  public static readonly DEFAULT_FILE_ENCODING = "utf-8";
-
-  public static readonly APP_SERVICE_PLAN = "App Service plan";
-  public static readonly AZURE_WEB_APP = "Azure Web App";
-  public static readonly BOT_CHANNEL_REGISTRATION = "Azure Bot Service channel registration";
-  public static readonly MS_TEAMS_CHANNEL = "Teams channel";
-  public static readonly AAD_APP = "AAD app";
-  public static readonly AAD_CLIENT_SECRET = "AAD client secret";
-  public static readonly APPSTUDIO_BOT_REGISTRATION = "App Studio bot registration";
-  public static readonly APPSTUDIO_MSG_ENDPOINT = "App Studio message endpoint";
-  public static readonly CONFIG_ITEM = "config item";
-  public static readonly SHORT_APP_NAME = "short app name";
```

Comment:
The error message is `'short app name' is missing`. I think maybe make `SHORT_APP_NAME` a unique key could be better than translate it.
For example:
`'appName' is missing.`